src/commands/responder.rs

```diff
@@ -31,6 +33,7 @@ impl CommandResponse {
             CommandResponse::Chat(uuid, msg) => CommandResponse::Chat(uuid, msg),
             CommandResponse::Activity(_, state) => CommandResponse::Activity(uuid, state),
             CommandResponse::RenameChat(_, name) => CommandResponse::RenameChat(uuid, name),
+            CommandResponse::RenameBranch(_, name) => CommandResponse::RenameBranch(uuid, name),
             CommandResponse::BackendMessage(_, msg) => CommandResponse::BackendMessage(uuid, msg),
```

Comment:
Cool, this works! I think past-this-PR later, It would be nice to fold the renames (and perhaps more), into a single message, that would remove a lot of code. This works great for now though!